locust/webui/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="/assets/favicon.ico" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
I think we fundamentally want to avoid using externally loaded resources, as much as it's tempting.

Lots of Locust users execute tests inside of private networks, so loading Google fonts will likely make things inconsistent.

We should statically cache whatever fonts we need, or use inbuilt ones IMO

Reply:
Unless it looks terrible, I think we should use the built in ones.